code/datums/traits/negative_quirk.dm

```diff
@@ -218,11 +217,24 @@
 			if(JOB_NAME_SHAFTMINER)
 				heirloom_type = pick(/obj/item/pickaxe/mini, /obj/item/shovel)
 
+	//When compounded with the above this makes a 33% chance for racial heirloom, even though this says prob(50)
+	if(prob(50))
+		if(islizard(H))
+			heirloom_type = /obj/item/heirloom_skull
+		if((ismoth(H)))
+			heirloom_type = /obj/item/flashlight/lantern/heirloom_moth
+		if(isipc(H))
+			heirloom_type = /obj/item/disk/tech_disk
+
+
+	//Generic heirloom, anywhere from 33% chance to 100% chance depending on race/job combination.
+	// Functions as a failsafe in the event that no other heirlooms were available, that's why this isn't a switch.
 	if(!heirloom_type)
```

Comment:
Shouldn't this be an `else if(prob(50))` and `else`? Right now there's a 50% chance to get a species-specific heirloom.

Reply:
Yes, it should be. Thank you